### PR TITLE
PRIME 1761 - Display review or renew title+message by determining if enrolment requires renewal or not

### PR DESCRIPTION
**Pull request by Paahn**
For ticket [1761](https://bcgovmoh.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=PRIME&modal=detail&selectedIssue=PRIME-1761)

- display renew title and message if enrolment expiration is within 90 days
- otherwise display review title and message

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts [hunk 1]
```diff
@@ -153,10 +155,18 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
...
+          // Determine if Enrolment is within 90 days of expiration
+          this.enrolmentWithin90DaysOfExpiry();
         })
       ).subscribe();
   }
 
+  public enrolmentWithin90DaysOfExpiry(): void {
+    const enrolment = this.enrolmentService.enrolment;
+    const expiryDate = enrolment.expiryDate;
+    this.withinDaysOfRenewal = DateUtils.withinDaysBeforeDate(expiryDate, 90) ? true : false;
```
Comment (johnnyl6) [inline prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts:167]: `withinDaysBeforeDate` already returns `boolean` so conditional operator is not needed.

**Review by AnaisHebert — APPROVED** [review]
duck

File: prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts [hunk 2]
```diff
@@ -153,10 +155,18 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
           this.enrolmentFormStateService.setForm(enrolment);
 
           this.enrolmentErrors = this.getEnrolmentErrors(enrolment);
+          // Determine if Enrolment is within 90 days of expiration
+          this.enrolmentWithin90DaysOfExpiry();
         })
       ).subscribe();
   }
 
+  public enrolmentWithin90DaysOfExpiry(): void {
```
Comment (johnnyl6) [inline prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts:164]: it seems `enrolmentWithin90DaysOfExpiry` is not called in the template so maybe only need `private` access modifier. other than this, LGTM

**Review by johnnyl6 — APPROVED** [review]
🚀

**Pull request merged**